qa/tasks/cephfs/test_misc.py

```diff
@@ -523,6 +523,80 @@ def test_session_ls(self):
 
     def test_client_ls(self):
         self._session_client_ls(['client', 'ls'])
+
+
+class TestSessionClientEvict(CephFSTestCase):
+    CLIENTS_REQUIRED = 3
```

Comment:
Uh oh - this with a yaml that configured 2 clients.

https://pulpito.ceph.com/vshankar-2023-12-06_15:12:46-fs-wip-vshankar-testing-20231206.125818-testing-default-smithi/7480482/

Reply:
IMO we can just remove the following link:

```
1-mds-2-client.yaml -> .qa/cephfs/clusters/1-mds-2-client.yaml
```
from

```
qa/suites/fs/multiclient/clusters/
```

Reply:
Addressed!